release-team/role-handbooks/branch-manager/README.md

```diff
@@ -1,235 +1,307 @@
+1. Participate in weekly 1hr release team meetings.
+1. Run `./branchff` on a daily basis as soon as the release branch i.e. `release-X.Y` [is created](#branch-creation).(Less than 45mins)
+1. Your time commitment increases during code freeze (when approaching the official release), expect 3 or more 1hr meetings in a week.
+1. Select shadows and guide them in preparation to become the section lead for the next cycle.
+1. Update this handbook where appropriate for the next release cycle.
+1. Have willingness to accommodate with different timezone esp. for the release team and your shadows.
+1. Participate in conversations that happen on [#sig-release] and [#release-managers]
+
+To get a better overview of the time it takes to run the release tools, you can take a look at the collected metrics from each of these [release cut issues][release-cut-issues] on the sig-release repo.
+
+[Branch Management diagram](https://docs.google.com/presentation/d/1usEbwHMSC8vR7HvbxHJBOj2ISdTkw9rmufQUq7fkIl4/edit?ts=5cc0de62#slide=id.g577153f745_1_405) illustrating the process over a 3 month [release lifecycle](https://docs.google.com/presentation/d/1usEbwHMSC8vR7HvbxHJBOj2ISdTkw9rmufQUq7fkIl4/edit?ts=5cc0de62#slide=id.g577153f745_3_0).
+
+[Kubernetes]: https://github.com/kubernetes/kubernetes/releases
+[release-1.15]: https://github.com/kubernetes/sig-release/tree/master/releases/release-1.15
+[#sig-release]: https://kubernetes.slack.com/messages/C2C40FMNF
+[#release-mangers]: https://kubernetes.slack.com/messages/CJH2GBF7Y
+[release-cut-issues]: https://github.com/kubernetes/sig-release/issues?utf8=%E2%9C%93&q=is%3Aissue+1.15.0+is%3Aclosed+
 
 ## Minimum Skills
 * Familiarity with basic Unix commands and able to debug shell scripts.
 * General knowledge of Google Cloud (Cloud Build and Cloud Storage).
 * Open to seeking help and can communicating clearly.
 
 # Prerequisite
-* Contact [Kubernetes Build Admins][kubernetes-build-admins], identifying yourself as the incoming release branch manager for the current release team and requesting to be added to the special privilege group for the GCP project used to build the releases
-   * TODO: should the release lead also request some type of read access here for themselves?
-   * TODO: should the branch manager shadows also get at least read access at the beginning?  If a shadow is going to exercise the process on behalf of the lead, they will need full read/write access ahead of time.
-   * TODO: can the group name be identified here in a clear/transparent way?
-* Ask the list owner(s) to give you access to **post** to these mailing lists:
-   * [kubernetes-announce][k-announce-list] via owner contact form [here][k-announce-request]
+A list of To Do(s) to get started as Branch Manager:
+
+Important:
+> - [ ] Request to be a [Kubernetes community org member](https://github.com/kubernetes/community/blob/master/community-membership.md#member).
```

Comment:
I think for this role particular role we should add org membership to the "Minimum Skills" list.  This is something achievable through shadowing the role, so it's not a high hurdle.  Given the set of privileges granted to those exercising this role I think it is completely reasonable to require folks to have demonstrated exposure to the project's workings and a track record of contribution.